content/news/2025-09-24-bevy-0.17/index.md

```diff
@@ -0,0 +1,1182 @@
+we're pleased to introduce "Feathers" - a comprehensive Bevy UI widget set. Feathers is meant to be Bevy's "developer tools" widget set, and it will be used to build the upcoming [Bevy Editor](https://bevy.org/news/bevys-fifth-birthday/#bevy-editor-design). It has a utilitarian look and feel, with an opinionated feature-set tailored for editors and graphical utilities. It builds on top of Bevy's new general-purpose "headless" widget set: `bevy_ui_widgets` (covered below). Feathers _can_ be used in games, but that is not its motivating use case.
+
+Feathers currently offers:
+
+- Standard widgets designed to match the look and feel of the planned Bevy Editor.
+- Components that can be leveraged to build custom editors, inspectors, and utility interfaces that feel consistent with other Bevy tooling.
+- Essential UI elements including buttons, sliders, checkboxes, menu buttons, and more.
+- Layout containers for organizing and structuring UI elements.
+- Decorative elements such as icons for visual enhancement.
+- Initial (simple / primitive) theme support to ensure consistent, configurable visual styling across applications. This is not the "final" Bevy UI theme system, but it provides some baseline functionality.
+- Accessibility features with built-in screen reader and assistive technology support.
+- Interactive cursor behavior that changes appropriately when hovering over widgets.
+- A virtual keyboard suitable for touchscreen text input.
+
+Feathers is still early in development. It is currently hidden behind the `experimental_bevy_feathers` feature flag. Feathers is still incomplete and likely to change in a variety of ways:
+
+- We will port Feathers to BSN (Bevy's [Next-Generation Scene/UI System](https://github.com/bevyengine/bevy/pull/20158/)) when that lands (targeted for **Bevy 0.18**).
+- We are still discussing the best way to handle UI callbacks / events in Feathers. It includes a proposal API, but the debate is ongoing!
+- We are still working on polishing up some UX issues.
+- There are missing widgets and features. Notably the "text input" widget is still being developed.
+
+If you're looking to experiment with building tooling for Bevy, enable it and take `feathers` for a test flight!
+Let us know what problems you run into, and feel free to contribute missing widgets and bugs upstream.
+
+If you can't wait to get your hands on `bevy_ui` widgets for your game,
+we recommend copying the Feathers code into your project and start hacking away at it!
+Feathers can serve as a helpful base to understand how to build and theme widgets in Bevy UI. It also illustrates how to use our new "headless" widget set: `bevy_ui_widgets`.
+
+## Realtime-filtered environment maps
+
+{{ heading_metadata(authors=["@mate-h"] prs=[19076, 20529]) }}
+(TODO: Embed screenshot of atmosphere-generated reflections)
+
+An environment map needs to be processed to be able to support uses beyond a simple skybox,
```

Comment:
Way too confusing imo, even for me. I would focus more on user use cases, and how this partially replaces the hard-to-build gltf-ibl-sampler stuff + EnvironmentMapLight, and enables more dynamic use cases like having procedural atmospheres contribute lighting to the scene instead of just being a background. Mention specific components by name.